src/Symfony/Bridge/Twig/Extension/LogoutUrlExtension.php

```diff
@@ -22,9 +22,9 @@
  */
 final class LogoutUrlExtension extends AbstractExtension
 {
-    private LogoutUrlGenerator $generator;
+    private LogoutUrlGeneratorInterface $generator;
 
-    public function __construct(LogoutUrlGenerator $generator)
+    public function __construct(LogoutUrlGeneratorInterface $generator)
```

Comment:
To keep compat with 6.4?
```suggestion
    public function __construct(LogoutUrlGeneratorInterface|LogoutUrlGenerator $generator)
```

Reply:
Good idea.